apps/style/applab/style.scss

```diff
@@ -8,10 +8,8 @@
 
 $root: '/blockly/media/applab/'; //TODO: Parameterize for asset pipeline
 
-// ** START OF RULES THAT ARE MODIFIED AT RUNTIME IN adjustAppSizeStyles() **
-
 div#visualization {
-  height: 400px;
+  height: 562.5px;
```

Comment:
Possibly. Does scss allow us to do that math in place? If not, I think even having a comment along these lines somewhere would be useful